test-server/package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
+    "react-pincode": "^1.0.2",
```

Comment:
@Rukmini-Meda there's no need to install the module in the test repo. Get rid of this line.